cmd/tuf/refresh.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/flynn/go-docopt"
+	"github.com/theupdateframework/go-tuf"
+)
+
+func init() {
+	register("refresh", cmdRefresh, `
+usage: tuf refresh [--snapshot-threshold=<hours> --timestamp-threshold=<hours>]
+
+Resign the snapshot and/or timestamp metadata if needed. If either metadata
+is expiring in less than the number of hours given by the threshold, its 
+metadata expiration will be refreshed and the change(s) will be committed.
+
+Alternatively, passphrases can be set via environment variables in the
+form of TUF_{{ROLE}}_PASSPHRASE
+
+Options:
+  --snapshot-threshold=<hours>   Set the threshold for when to resign expiring
```

Comment:
the wording for threshold is overloaded with the threshold for signing, maybe this could be `--snapshot-time` or `--snapshot-hours`? I'm not sure.

Please clarify in this option that this is in hour units